src/Makefile.am

```diff
@@ -129,15 +129,17 @@ $(desktop_in_files): %.desktop.in: %.desktop.in.in Makefile
 desktopdir=$(datadir)/applications
 desktop_DATA = $(desktop_in_files:.desktop.in=.desktop)
 
-@INTLTOOL_DESKTOP_RULE@
+$(desktop_DATA): $(addsuffix .in, $(desktop_DATA))
+	cp -f $< $@
```

Comment:
I wonder if this is sufficient. If *.desktop.in don't contain translatable strings, that would be; otherwise we need to use `msgfmt --desktop` to merge the translations:
https://www.gnu.org/software/gettext/manual/html_node/msgfmt-Invocation.html#Desktop-Entry-mode-operations